specification/electronic-prescription-service-api.yaml

```diff
@@ -377,6 +547,61 @@ paths:
                   A valid request to return a prescription.
                 value:
                   $ref: components/examples/secondary-care/homecare/acute/nominated-pharmacy/clinical-practitioner/2-Task-Request-Return-200_OK.json
+      responses:
+        '200':
+          description: Action completed successfully.
+          content:
+            application/fhir+json:
+              schema:
+                $ref: '#/components/responses/200OperationOutcome'
+              examples:
+                OperationOutcome:
+                  description: |
+                    A successful response to a return request.
+                  value:
+                    $ref: components/examples/operation-outcome-success.json
+        '4XX':
+          description: |
+            An error occurred as follows:
+
+            | HTTP status | Error code  | Description      |
+            | ----------- | ----------- | ---------------- |
+            | 400         | BAD_REQUEST | Invalid request. |
+          content:
+            application/fhir+json:
+              examples:
+                example:
+                  description: |
+                    An error response to a return request.
+                  value:
+                    $ref: components/examples/release/not-found-error.json
+  /FHIR/R4/Task#withdraw:
+    post:
+      operationId: withdraw
+      summary: Withdraw a dispense-notification message
```

Comment:
maybe get sachin's opinion on these summaries cause i think it can be difficult to sum up what the endpoint does without using overly specific wording?

Reply:
Will do